streams/core.py

```diff
@@ -202,13 +202,13 @@ def delay(self, interval, loop=None):
         """ Add a time delay to results """
         return delay(interval, self, loop=None)
 
-    def combine_latest(self, *others):
+    def combine_latest(self, *others, **kwargs):
```

Comment:
I would put `emit_on=None)` instead to be more explicit. It might help with documentation and propagating down what is used.